cou/steps/hypervisor.py

```diff
@@ -180,11 +180,10 @@ def _upgrade_plan_sanity_checks(
                 )
                 continue
 
-            units = group.app_units[app.name]
-            logger.info("running sanity checks for %s units of %s app", app.name, units)
+            logger.info("running sanity checks for %s app", app.name)
             # Note(rgildein): We don't catch the error here because we shouldn't generate any
             #                 update plan if sanity checks for any application fails.
-            app.upgrade_plan_sanity_checks(target, units)
```

Comment:
isn't it used here?

Reply:
No, it's removed.

Reply:
https://github.com/canonical/charmed-openstack-upgrader/pull/582/files#diff-cc23959811834c0e219a14e62eeb8f126146f7f465644751cae5ddb6f84ee318L939-L940 

maybe this fool the static code checker that the arg was used

Reply:
Ah yes, I was wondering that, and just checked now - it was indeed fooling pylint.